doc/release-process.md

```diff
@@ -7,7 +7,9 @@ The current release process for weave gitops is fairly straightforward. You need
 - Add a record of the new version in the checkpoint system
 
 # Creating the release
-- Update the package.json `version` field to reflect the new version. This change must be on `main` before creating a release.
+- Update `package.json` and `package-lock.json` . These changes must be on `main` before creating a release.
+  - Update the package.json `version` field to reflect the new version.
+  - Run `npm ci` to update the file `package-lock.json`.
```

Comment:
Let's change this to say `edit the package-lock.json manually with the new version`. I think this caused issues last time.

Reply:
I thought we needed to add `npm ci` to this process to avoid issues related to differences in the file `package-lock.json` in the release workflow 🤔

Reply:
I don't  believe `npm ci` will edit the `package-lock.json`. I will have to check.

Reply:
That's how we mitigated the differences when we did the release. Unless this https://github.com/weaveworks/weave-gitops/pull/743/files helps to avoid the need to run `npm ci`